src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.test.ts

```diff
@@ -319,7 +319,21 @@ describe("verify", () => {
     });
     it("should fail when corresponding proof to key is not found in proof", async () => {
       whenPublicKeyResolvesSuccessfully();
-      const res = await openAttestationDidSignedDocumentStatus.verify({ ...documentDidSigned, proof: [] }, options);
+      const res = await openAttestationDidSignedDocumentStatus.verify(
+        {
+          ...documentDidSigned,
+          proof: [
+            {
+              created: "2021-03-25T07:52:31.291Z",
+              type: "OpenAttestationSignature2018",
+              proofPurpose: "assertionMethod",
+              verificationMethod: "did:ethr:0xdede",
+              signature: "0xabcf",
```

Comment:
i'm slightly surprised this doesn't fail for invalid length?

Reply:
The version 2 document only check the required object in proof exist and the data type doesn't check the length.

Reply:
There is no length check on the signature indeed